### PR TITLE
Matrix multiplication and execution time recording

### PR DESCRIPTION
**Pull request by sendjasni**
Matrix multiplication with execution time recording using the cpu CLOCK_REALTIME

### CODE REVIEW

File: math/Matrix Multiplication/matmul.c [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ * Matrix multiplication with execution time recording  
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+
+const int MAT_SIZE = 500;
+
+void MatrixMultiplication()
```
Comment (christianbender) [inline math/Matrix Multiplication/matmul.c]: * Write this function only for matrix-multiplication. Not more!
* Write tests in the main-function

Comment (sendjasni) [reply math/Matrix Multiplication/matmul.c]: Ok, thanks for the reply, I'll do that.

**Comment by sendjasni** [comment]
I updated the matmul code.

**Comment by christianbender** [comment]
@sendjasni Thanks I'll look it at

**Review by abranhe — APPROVED** [review]

**Pull request merged**